client/src/views/Create.vue

```diff
@@ -142,10 +142,10 @@ export default {
           component: CosmologyForm,
           model: 'cosmo',
         },
-        {
-          component: HaloModelForm,
-          model: 'halo_model',
-        },
+        // {
+        //   component: HaloModelForm,
+        //   model: 'halo_model',
+        // },
```

Comment:
Why was this one removed?

Reply:
It crashes the server right now

Reply:
It doesn't seem to be crashing anything when running it locally with that included. Does doing something specific cause it to crash?